README.md

```diff
@@ -1,10 +1,47 @@
-# Northcoders News API
+# Northcoders Mior News API
 
-## These files need to be created to connect to databases
+## Link to the hosted version
 
-.env.test -> to connect to test database
+https://be-nc-news-mior.onrender.com/
+
+## Summary of project
+
+The project is to build an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as Reddit) which should provide this information to the front end architecture.
+
+## Instruction
+
+### Cloning the repository
+
+Repository can be clone from this URL:
+https://github.com/miors/be-nc-news-mior
+
+Depending on your settings, to clone, either type -> `git clone https://github.com/miors/be-nc-news-mior.git`
+or
+type -> `git clone git@github.com:miors/be-nc-news-mior.git`
+
+### Install dependencies
+
+Type-> `npm install`
+
+### Seed database
+
```

Comment:
we need .env files too before we seed!
what about setting up the database? how do we do that?